cle/backends/java/apk.py

```diff
@@ -53,15 +60,13 @@ def __init__(self, apk_path, binary_stream, entry_point=None, entry_point_params
         else:
             l.info("Using user defined JNI lib(s) %s (load path(s) %s)", jni_libs, jni_libs_ld_path)
 
-        if not entry_point:
-            try:
-                from pyaxmlparser import APK as APKParser
-                apk_parser = APKParser(apk_path)
+        if PYAXMLPARSER_INSTALLED:
+            apk_parser = APKParser(apk_path)
+
+            if not entry_point:
```

Comment:
You are changing the logic. In the old days, when `entry_point` is not specified and when `pyaxmlparser` is not installed, it will raise an `ImportError`. But after your change, if `pyaxmlparser` is not installed, it will accept unspecified `entrypoint` (and `main_activity` as well as `entry_point_params`). The original logic is more reasonable. Otherwise `entrypoint` will be passed to `super().__init__()` without proper assignment.

Reply:
I resolved the condition that ```entrypoint``` can be unspecified. ```apk parser``` will be generated before.